hosts

```diff
@@ -37,6 +34,7 @@ example-ca ansible_ssh_host=127.0.0.1 ansible_ssh_port=22 ansible_release_host=1
 [capture-agents-datapath-usb]
 
 [capture-agents-dual-datapath-split]
+vision.csi.cam.ac.uk ansible_ssh_host=131.111.56.116 ansible_ssh_port=22 ansible_release_host=127.0.0.1 ansible_uom_hostname=vision.csi.cam.ac.uk support_group=example-support home_tunnel_port=22 nagios_tunnel_port=None deploy_environment=staging name_mask=None ansible_failovermic=FALSE CA_interactive=None oc_default_series=None oc_default_email=None vl_public=false blinkstick=False vapix_inverted=true ptz_tilt_lock=false vl_private_ep_acl=False cam1_host=192.168.0.90 cam2_host=None CA_interactive_role=example-role
```

Comment:
I'd leave the example comment and add this line below it. This may also help with the conflict with master.